CHANGELOG.md

```diff
@@ -42,7 +42,7 @@
 
 ## v1.14.4 (2020-03-25)
     Fix fips-mode check for RHEL8 boxes
-    Fix potential CVE in tarfile w/ symlink
+    Fix CVE-2020-10696 in tarfile w/ symlink
```

Comment:
I'd rather not change the initial wording.  The reason being is this wording comes directly from the commit title and I at least use them from looking up particular commits later on when trying to determine what landed in other versions of projects that include Buildah.  But that said, I'd be fine with appending something.  Perhaps:
`Fix potential CVE in tarfile w/ symlink (Edit 02-Jun-2020: Addresses CVE-2020-10696)`

Could I also prevail on you to make a similar change to changelog.txt?

FWIW, we try to include the CVE number in the commit titles, but obviously failed on this one.